Test main - settings/avatar - visit gravatar with no gravatar set FAILED

This test has failed on latest with ff18, ff-beta, and with all ff flavours on stage, sometimes with `AssertionError: has error text` and sometimes with an "element not found".

```
[03:41:47][Step 1/1] AssertionError: has error text
[03:41:47][Step 1/1]   at Function.assert.ok  <node_modules/intern-geezer/assert.js:525:10>
[03:41:47][Step 1/1]   at CompatCommand.<anonymous>  <tests/functional/avatar.js:242:20>
[03:41:47][Step 1/1]   at runCallback  <node_modules/intern-geezer/node_modules/leadfoot/Command.js:509:31>
[03:41:47][Step 1/1]   at CompatCommand.<anonymous>  <node_modules/intern-geezer/node_modules/leadfoot/Command.js:526:11>
[03:41:47][Step 1/1]   at <node_modules/intern-geezer/node_modules/leadfoot/node_modules/dojo/Promise.js:256:37>
[03:41:47][Step 1/1]   at runCallbacks  <node_modules/intern-geezer/node_modules/leadfoot/node_modules/dojo/Promise.js:13:18>
[03:41:47][Step 1/1]   at <node_modules/intern-geezer/node_modules/leadfoot/node_modules/dojo/Promise.js:204:17>
[03:41:47][Step 1/1]   at Object.run [as _onImmediate]  <node_modules/intern-geezer/node_modules/leadfoot/node_modules/dojo/Promise.js:143:29>
[03:41:47][Step 1/1]   at processImmediate [as _immediateCallback]  <timers.js:354:15>
[03:41:47][Step 1/1]   at new CompatCommand  <node_modules/intern-geezer/runner.js:212:14>
[03:41:47][Step 1/1]   at CompatCommand.Command.then  <node_modules/intern-geezer/node_modules/leadfoot/Command.js:525:10>
[03:41:47][Step 1/1]   at registerSuite.visit gravatar with no gravatar set [as test]  <tests/functional/avatar.js:241:12>
[03:41:47][Step 1/1]   at declare.run  <node_modules/intern-geezer/lib/Test.js:166:19>
[03:41:47][Step 1/1]   at <node_modules/intern-geezer/lib/Suite.js:232:13>
[03:41:47][Step 1/1]   at signalListener  <node_modules/intern-geezer/node_modules/dojo/Deferred.js:37:21>
[03:41:47][Step 1/1]   at signalWaiting  <node_modules/intern-geezer/node_modules/dojo/Deferred.js:28:4>
[03:41:47][Step 1/1]   at resolve  <node_modules/intern-geezer/node_modules/dojo/Deferred.js:192:5>
[03:41:47][Step 1/1]   at runNextSuite  <node_modules/intern-geezer/lib/Suite.js:196:50>
[03:41:47][Step 1/1]   at signalListener  <node_modules/intern-geezer/node_modules/dojo/Deferred.js:37:21>
[03:41:47][Step 1/1]   at new CompatCommand  <node_modules/intern-geezer/runner.js:212:14>
[03:41:47][Step 1/1]   at CompatCommand.Command.end  <node_modules/intern-geezer/node_modules/leadfoot/Command.js:475:10>
[03:41:47][Step 1/1]   at registerSuite.visit gravatar with no gravatar set [as test]  <tests/functional/avatar.js:244:10>
[03:41:47][Step 1/1]   at declare.run  <node_modules/intern-geezer/lib/Test.js:166:19>
[03:41:47][Step 1/1]   at <node_modules/intern-geezer/lib/Suite.js:232:13>
[03:41:47][Step 1/1]   at signalListener  <node_modules/intern-geezer/node_modules/dojo/Deferred.js:37:21>
[03:41:47][Step 1/1]   at signalWaiting  <node_modules/intern-geezer/node_modules/dojo/Deferred.js:28:4>
[03:41:47][Step 1/1]   at resolve  <node_modules/intern-geezer/node_modules/dojo/Deferred.js:192:5>
[03:41:47][Step 1/1]   at runNextSuite  <node_modules/intern-geezer/lib/Suite.js:196:50>
[03:41:47][Step 1/1]   at signalListener  <node_modules/intern-geezer/node_modules/dojo/Deferred.js:37:21>
```

--- a/tests/functional/avatar.js
+++ b/tests/functional/avatar.js
@@ -236,6 +236,7 @@ define([
         .end()
 
         // success is seeing the error text
+        .then(FunctionalHelpers.visibleByQSA('.error'))
         .findByCssSelector('.error')
           .getVisibleText()
           .then(function (val) {

